Makefile

```diff
@@ -1,8 +1,14 @@
-DOCKER := docker run --rm -v $(shell pwd)/../../../..:/go
+include common.mk
 
-GATEKEEPER := dist/gatekeeper
-
-all: $(GATEKEEPER)
+all: $(RPM)
 
 $(GATEKEEPER) : cmd/gatekeeper/main.go
-	$(DOCKER) gatekeeper-build go build -ldflags "-X main.BuildTime=`date -u '+%Y-%m-%d_%I:%M:%S%p'` -X main.Version=`git -C ./ describe --abbrev=0 --tags HEAD`" -a -installsuffix cgo -o src/github.com/nemosupremo/vault-gatekeeper/dist/gatekeeper github.com/nemosupremo/vault-gatekeeper/cmd/gatekeeper
+	$(DOCKER) docker.br.hmheng.io/hmheng-infra/gatekeeper-build:v1.0.3-10-ga65b145 make -C /go/src/github.com/nemosupremo/vault-gatekeeper -f Makefile.docker $@
+
+$(RPM): $(GATEKEEPER)
+	$(DOCKER) docker.br.hmheng.io/hmheng-infra/fpm:latest make -C /go/src/github.com/nemosupremo/vault-gatekeeper -f Makefile.docker $@
```

Comment:
There's the internal repo again. Also we might want to simplify the build with a multistage-build?